ui/app/styles/components/console-ui-panel.scss

```diff
@@ -125,6 +125,10 @@ $console-close-height: 35px;
   min-height: 400px;
 }
 
+.main--console-open {
```

Comment:
🙈 this a great css pattern that we learned from the offsite! should we use this as a standard pattern for the future? https://css-tricks.com/bem-101/